Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -3385,42 +3385,98 @@ def get_machine_data(machine):
     return machine_data
 
 
-def get_file_statistics_command(client: MsClient, args: dict):
+def get_file_statistics_command(client: MsClient, args: dict) -> CommandResults:
     """Retrieves the statistics on the given file.
 
     Returns:
-        (str, dict, dict). Human readable, context, raw response
+        CommandResults.
     """
-    file_sha1 = args.get('file_hash')
+    file_sha1 = args.get('file_hash', '')
     response = client.get_file_statistics(file_sha1)
     file_stat = get_file_statistics_context(response)
-    human_readable = tableToMarkdown(f'Statistics on {file_sha1} file:', file_stat, removeNull=True)
+    human_readable = get_file_statistics_human_readable(file_sha1, response)
     context_output = {
         'Sha1': file_sha1,
         'Statistics': file_stat
     }
-    entry_context = {
-        'MicrosoftATP.FileStatistics(val.Sha1 === obj.Sha1)': context_output
+    file_indicator = get_file_statistics_indicator(file_sha1, response)
+    demisto.debug(f'Got file indicator: {file_indicator.to_context()}')
+
+    return CommandResults(
+        outputs_prefix='MicrosoftATP.FileStatistics',
+        outputs_key_field='Sha1',
+        indicator=file_indicator,
+        readable_output=human_readable,
+        outputs=context_output,
+        raw_response=response,
+    )
+
+
+def get_file_statistics_human_readable(file_hash: str, file_stat_response: dict) -> str:
+    """Gets the file statistics response and returns as a human readable markdown table.
+
+    Args:
+        file_hash (str): Hash of file
+        file_stat_response (dict): File statistics response body
+
+    Returns:
+        (dict). File statistics context
+    """
+    formatted_data = {
+        'Global Prevalence': file_stat_response.get('globallyPrevalence'),
```

Comment:
add a comment about the `ly` part

Reply:
I have added a comment a few lines down in commit f169d032069933d5959313291c693b7163970c9a

Let me know if any further changes are required in this area